src/test/javascript/spec/component/shared/metis/postings-footer/post-footer/post-footer.component.spec.ts

```diff
@@ -1,67 +1,123 @@
 import * as chai from 'chai';
 import * as sinonChai from 'sinon-chai';
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-import { Post } from 'app/entities/metis/post.model';
 import { PostFooterComponent } from 'app/shared/metis/postings-footer/post-footer/post-footer.component';
-import { By } from '@angular/platform-browser';
 import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
-import { MockComponent, MockPipe } from 'ng-mocks';
+import { MockComponent, MockModule, MockPipe } from 'ng-mocks';
 import { FaIconComponent } from '@fortawesome/angular-fontawesome';
 import { PostReactionsBarComponent } from 'app/shared/metis/postings-reactions-bar/post-reactions-bar/post-reactions-bar.component';
+import { ArtemisCoursesRoutingModule } from 'app/overview/courses-routing.module';
+import { MetisService } from 'app/shared/metis/metis.service';
+import { MockMetisService } from '../../../../../helpers/mocks/service/mock-metis-service.service';
+import { SinonStub, stub } from 'sinon';
+import { MockTranslateService } from '../../../../../helpers/mocks/service/mock-translate.service';
+import { TranslateService } from '@ngx-translate/core';
+import { PageType } from 'app/shared/metis/metis.util';
+import { getElement, getElements } from '../../../../../helpers/utils/general.utils';
+import {
+    metisLecture,
+    metisPostExerciseUser1,
+    metisPostLectureUser1,
+    metisPostLectureUser2,
+    metisPostOrganization,
+    metisPostTechSupport,
+    metisTags,
+} from '../../../../../helpers/sample/metis-sample-data';
 
 chai.use(sinonChai);
 const expect = chai.expect;
 
 describe('PostFooterComponent', () => {
     let component: PostFooterComponent;
     let fixture: ComponentFixture<PostFooterComponent>;
-
-    const initialTags = ['tag1', 'tag2'];
+    let metisService: MetisService;
+    let metisServiceGetPageTypeStub: SinonStub;
     const updatedTags = ['tag1', 'tag2', 'tag3'];
 
-    const post = {
-        id: 1,
-        tags: initialTags,
-    } as Post;
-
     beforeEach(() => {
         return TestBed.configureTestingModule({
+            imports: [MockModule(ArtemisCoursesRoutingModule)],
+            providers: [
+                { provide: MetisService, useClass: MockMetisService },
+                {
```

Comment:
Prettier does not complain, so it has to be correct, but I feel like it should be in one line (This is really not at all important, if you want you could try to compact it and see if it gets broken again, but you can also just ignore it)

Reply:
I changed it :)